crates/infra/cli/src/commands/setup/workspace/mod.rs

```diff
@@ -1,18 +0,0 @@
-use anyhow::Result;
-use infra_utils::{commands::Command, github::GitHub};
-
-pub fn setup_workspace() -> Result<()> {
-    if !GitHub::is_running_in_ci() {
-        warm_up_ide_tools()?;
-    }
-
-    Ok(())
-}
-
-/// Warm up IDE tools in case it was a fresh installation.
-fn warm_up_ide_tools() -> Result<()> {
-    Command::new("rust-analyzer").flag("--version").run()?;
```

Comment:
That was only needed since it was pulled by Hermit and not rustup itself, right?

Reply:
Gotcha, thanks for the explanation!